queue/driver_mongo.go

```diff
@@ -1400,14 +1268,146 @@ type dispatchAttemptInfo struct {
 	// misses is the number of times the dispatcher attempted to dispatch a job
 	// but failed.
 	misses int
+	// total is the total number of documents the cursor returned.
```

Comment:
total is the total number of jobs that were considered for dispatch.

Reply:
total went away